Tools/CMake/Modules.cmake

```diff
@@ -150,12 +150,14 @@ add_custom_target(
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/R-Interface
   DEPENDS ${MODULES_BINARY_PATH}/jaspBase-installed-successfully.log
           ${MODULES_BINARY_PATH}/jaspGraphs-installed-successfully.log
-          ${MODULES_BINARY_PATH}/jaspTools-installed-successfully.log)
+          )
 
 # This happens during the configuration!
 file(
   WRITE ${MODULES_RENV_ROOT_PATH}/install-jaspBase.R
   "
+    .libPaths('${${R_LIBRARY_PATH}}') # make sure to only look in our local library
```

Comment:
Good point, but `${${R_LIBRARY_PATH}}` should be `${R_LIBRARY_PATH}` unless I'm missing something.

Reply:
Oops

Reply:
Should I fix it here or are you doing it in your followup branch?

Reply:
Weird, but I guess you fixed it and github showed me a bunch of outdated commits

Reply:
Yes, I fixed a few things already!